### ISSUE TITLE
Error when trying to process frames

### ISSUE THREAD

**Issue by HandsomeDevilv112** [samim23/DeepDreamAnim#8]
First up, big fan of your work, and I'm very excited about getting this working.

When I try to actually process the frames (I believe the command line should look something like) "C:\Anaconda>python dreamer.py --input frames\bp --output pic" it goes through some parts of the operation and then fails.

Traceback (most recent call last):
  File "dreamer.py", line 339, in <module>
    args.zoom, args.stepsize, args.blend, args.layers, args.guide, args.gpu, args.flow)
  File "dreamer.py", line 166, in main
    img = PIL.Image.open(inputdir + '/' + vids[0])
IndexError: list index out of range

As I understand it, "indexerror: out of range" is that it usually is in reference to a line that isn't there, but line 339 is totally there, and that appears to be the bug reported in https://github.com/samim23/DeepDreamAnim/issues/3 , and the closing comment says that the index error is an image processing the last frame. Which is odd, because it isn't processing any frames at the moment.

The research I've done about the pil error has not turned up an answer that I've been able to put to solving the problem. 
I've been talking to a friend who has had success with running deepdreamanim, he said that it could be related to my caffe paths, but as far as I can tell, he and I are using the same pathing.
Lines 134-138    
# Loading DNN model

```
model_name = 'bvlc_googlenet'
model_path = 'C:/anaconda/caffe/models/' + model_name + '/'
net_fn = model_path + 'deploy.prototxt'
param_fn = model_path + 'bvlc_googlenet.caffemodel'
```

I've also tried adding a 0 before each file name, but have not tried the files without the zero. 
Any and all help is appreciated. Thanks.

**Comment by samim23** [comment]
Try extracting the frames of a video with the lib as described: 
'python dreamer.py --input myvideo/video.mp4 --output myvideo --extract 1'

It will generate a specific filename convention like this:
frame_000041.png

If you want to use jpg´s, change .png in dreamer.py everywhere to .jpg.  

as well double check your paths (image path & model path)

hope that helps
s

**Comment by HandsomeDevilv112** [comment]
I'm having difficulty working with ffmpeg. I have tried both the linux and windows versions being in my anaconda folder, and I have not had any luck with either. Though I feel like I'm missing something basic.I'm currently receiving "FFmpeg is not recognized as an internal or external command"
I do currently have the target frames in jpg, which did not occur to me as an issue at the time. Thank you for catching that.
Changing the naming convention does allow me to process the files through deepdreamanim.
Closing this issue as "a problem located between the keyboard and the chair" and I will do further research on my problems with Ffmpeg separately from this issue. 
Thank you.
